moses/baselines/ngram/ngram.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import moses
+from moses import CharVocab
+from tqdm import tqdm
```

Comment:
Use `from tqdm.auto import tqdm`. In this case, tqdm will automatically choose which version to use — Jupyter or console